### PR TITLE
Results page with utilities

### PR DESCRIPTION
**Pull request by Adamya05**

### CODE REVIEW

**Review by RonanSandoval — DISMISSED** [review]
Add `transient` to location in QRCode

**Review by RTHaskins — CHANGES_REQUESTED** [review]
Delete changes to misc.xml file as they are specific to your computer

File: QR-Rangers/app/src/main/java/com/example/qr_rangers/QRCode.java [hunk 1]
```diff
@@ -115,7 +118,7 @@ public boolean equals(Object code){
         if(!(code instanceof QRCode)){
             throw new IllegalArgumentException();
         }
-        return this.codeInfo == ((QRCode) code).codeInfo; // Possibly change to .equals() depending on what codeInfo ends up being
+        return this.codeInfo.equals(((QRCode) code).codeInfo); // Possibly change to .equals() depending on what codeInfo ends up being
```
Comment (AttacktoWin) [inline QR-Rangers/app/src/main/java/com/example/qr_rangers/QRCode.java]: You can get rid of this comment
```suggestion
        return this.codeInfo.equals(((QRCode) code).codeInfo);
```

**Review by AttacktoWin — APPROVED** [review]
Looks good, the only things to change would be that comment (not something that needs to change right now wither way), and removing changes to the generated files that Ryan mentioned in the discord

**Review by RTHaskins — APPROVED** [review]

**Pull request merged**